app.fotema.Fotema.json

```diff
@@ -0,0 +1,207 @@
+{
+  "id": "app.fotema.Fotema",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.llvm18"
+  ],
+  "command": "fotema",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures:ro",
+    "--env=RUST_BACKTRACE=1"
```

Comment:
Do you need the logs? Seems it is better to turn this off to avoid log spam.

Reply:
Hmm... I agree that the 1.0.0 release is logging too much. However, to fix that properly I think I need to change some of the logging configuration in the code and cut a new release.

Reply:
This is a minor issue, you can fix this post merge.